components/blocks/nav/SiteNavbar/SiteNavbar.vue

```diff
@@ -102,13 +102,15 @@ function logout() {
       <BaseModal v-show="state.showModal" @close="state.showModal = false">
         <LogInCard
           v-show="state.showLogin"
+          data-testId="log-in-card"
           class="shadow-xl"
           :modal="true"
           @close="state.showModal = false"
           @sign-up-click="state.showLogin = false"
         />
         <SignUpCard
           v-show="!state.showLogin"
+          data-testId="sign-up-card"
```

Comment:
Nit: seen that all `data-testid`s are cased that way, but here, they're `data-testId`. We should standardise this

Reply:
ya let me rewrite all to `data-testId` just because it'll be a bit easier to read but ya should be uniform.

Reply:
nvm i'll set it to `data-testid` as that's what i see in docs.